lntest/itest/lnd_test.go

```diff
@@ -3176,6 +3176,9 @@ func channelForceClosureTest(net *lntest.NetworkHarness, t *harnessTest,
 	ctxt, _ := context.WithTimeout(ctxb, defaultTimeout)
 	net.ConnectNodes(ctxt, t.t, alice, carol)
 
+	// We need one additional UTXO for sweeping the remote anchor.
+	net.SendCoins(ctxt, t.t, btcutil.SatoshiPerBitcoin, alice)
```

Comment:
Ah yeah, this is something we considered as a short term fix. I think it works for now, but we'll need to gague to see if in practice the current logic w.r.t the sweeper and anchor outputs exclusive groups is problematic or not. I think @wpaulino has a more fundamental solution, but this should give us a bit more breathing room.